doc/README.Win32.md

```diff
@@ -174,6 +174,18 @@ with control-S.
 Visual Studio will then re-run CMake.  If that completes without errors,
 you can build with Build > "Build All".
 
+Building libpcap - Using vcpkg
+
+You can download and install libpcap using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install libpcap
+
+The libpcap port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
```

Comment:
Microsoft has enough resources to make sure their package of this library is up to date, is that correct? (yes/no)

This pull request will be closed in 7 days if the review stays abandoned.

Reply:
Yes, If a new version of libpcap is released and some users need to update, we will update the version in the vcpkg warehouse to the latest version soon.

Reply:
Thank you for explaining. What you say above is "Microsoft has enough resources to keep their package up to date. Microsoft will not keep their package up to date. Instead the package users will have to monitor the package freshness."

This answers this question.